src/client/components/forms/creator.js

```diff
@@ -111,6 +137,12 @@ class CreatorForm extends React.Component {
 			});
 	}
 
+	/**
+	* Renders the document and allows the author to submit the data
+	* Includes a loading spinner if there is delay in data display due to
+	* connectivity.
+	* @return {HTML} a HTML document which displays a creatoreditor
```

Comment:
Actually returns a React Element - let's use the `ReactElement` type to indicate this from now on.